ui/perfherder/RevisionInformation.jsx

```diff
@@ -11,7 +11,7 @@ function getRevisionSpecificDetails(
   project,
   isBaseline,
   resultSet,
-  selectedTimeRange = null,
+  selectedTimeRange = undefined,
```

Comment:
Did you figure out the cause of the 'revision is undefined' error in this function?

If not, it might be worth doing a boolean check for revision before calling this function in case that was an edge case.

Reply:
Yes, I was missing a boolean check for the last ListGroupItem. Thanks for noticing this!